Ntust_Cloud_System/Ntust_Cloud_System/FileOperator/FileOperator.h

```diff
@@ -0,0 +1,30 @@
+//
+//  FileOperator.h
+//  Ntust_Cloud_System
+//
+//  Created by 林仁鴻 on 2021/4/13.
+//
+
+
+#ifndef FileOperator_h
+#define FileOperator_h
+
+#include "ResponseHandler.h"
+#include <string>
+using namespace std;
+
+class FileOperator
+{
+    
+public:
+    FileOperator();
+    ~FileOperator();
+    int Upload(const string &, string);
+    int Download(const string &);
```

Comment:
Please add comments for each method. See:
https://google.github.io/styleguide/cppguide.html#Function_Comments